integrations/basecamp/nango.yaml

```diff
@@ -15,9 +15,17 @@ integrations:
                     path: /projects
                     group: Projects
                 output: BasecampProjectsResponse
-
+            fetch-todolists:
+                description: |
+                  Fetch all todolists in a project. Example of an input Example: `{  projectId: 1234, todoSetId: 9999  }.Fetch your projects via the fetch-projects action, then locate the project's dock item where "name": "todoset". The id there is your todoSetId.
```

Comment:
I don’t think we need to include the JSON example in our in the description. That’s what the input in the yaml is for and also why the automated documentation exists both in the readme and in the api spec for the action